sentinelsat/scripts/cli.py

```diff
@@ -153,12 +154,14 @@ def cli(
     url,
     order_by,
     limit,
+    dhusversion,
 ):
     """Search for Sentinel products and, optionally, download all the results
     and/or create a geojson file with the search result footprints.
     Beyond your Copernicus Open Access Hub user and password, you must pass a geojson file
     containing the geometry of the area you want to search for or the UUIDs of the products. If you
     don't specify the start and end dates, it will search in the last 24 hours.
+    Can also provide the Copernicus Open Access Hub (DHuS) version.
```

Comment:
Can be removed from the docstring - the help text of the click decorator is sufficient info.

Reply:
Removing.